### PR TITLE
Perform Delaware precinct county assignments

### LINKED ISSUE #163: Delaware precincts have no county assignment
Delaware's precincts are all assigned to county `000`, which does not exist.

### PR DESCRIPTION
**Pull request by sigpwned**
Closes #163.

Note that this PR depends on the docker updates in #162, so it includes PR #162 as well. Please review #162 first, and if it's acceptable merge it before reviewing and merging this PR.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -334,14 +363,45 @@ out/18-indiana/state.gpkg: data/18-indiana/statewide/2010/tl_2012_18_vtd10.zip \
 
 	rm -rf 'out/18-indiana/source'
 
-out/19-iowa/state.gpkg: data/19-iowa/statewide/2016/pcts_04172014_0908am.zip data/template.shp
+out/19-iowa/state.gpkg: data/19-iowa/statewide/2016/pcts_04172014_0908am.zip data/19-iowa/counties/tl_2016_19_cousub.zip data/template.shp
```
[the remark this answers is not in the document]
Comment (sigpwned) [reply Makefile]: Fixed in further commits to #162.

File: data/10-delaware/README.md [hunk 2]
```diff
@@ -36,3 +38,9 @@
   - Precincts 17-02, 16-31, and 16-41 are the reporting units for UOCAVA votes (for New Castle, Kent, and Sussex Counties, respectively). These have no actual geography and the votes reported were not distributed to other precincts.
   - Precinct 13-02 appears in the shapefile, but not in election returns. However, the POPULATION column which came supplied in the shapefile reports 0 population.
   - Though the shapefile is labeled as the 2012 version, personal communication with the state office confirms that these will be in place throughout the decade.
+
+## Additional Boundary Geodata
+
+- **2016** Delaware county outlines pulled from [TIGER census project](https://catalog.data.gov/dataset/tiger-line-shapefile-2016-state-delaware-current-county-subdivision-state-based)
+  - Note that TIGER file contains historical court district outlines, which roll up to counties.
```
[the remark this answers is not in the document]
Comment (sigpwned) [reply data/10-delaware/README.md]: Ha! What's so strange about it? It looks normal to me, but I've been staring at a screen all evening! :)

[the remark this answers is not in the document]
Comment (nvkelso) [reply data/10-delaware/README.md]: `localadmin` level geography (below county, and really even the `county` level) in the USA is anything but conventional :)

[the remark this answers is not in the document]
Comment (sigpwned) [reply data/10-delaware/README.md]: Ah! I see. And yes, I agree! I was very surprised to find that the "county" file doesn't really contain counties. Silly me, thinking that a filename from the census would correspond directly to the file's contents.

**Pull request merged**